### PR TITLE
completed  AllCommodities events

### PR DESCRIPTION
**Pull request by its-quan-ng**

### CODE REVIEW

File: PharmacyManagement/Program.cs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+using PharmacyManagement.Commodity;
+using PharmacyManagement.HumanManage;
```
Comment (ngocloan202) [inline PharmacyManagement/Program.cs]: delete it

File: PharmacyManagement/Commodity/AllCommodities.cs [hunk 2]
```diff
@@ -1,21 +1,155 @@
...
+        }
+
+        private void UpdateCommodity()
+        {
+            string updateCommodityQuery = @"UPDATE COMMODITY
+                                   SET CommodityName = @newCommodityName,
+                                       Manufacturer = @Manufacturer,
+                                       Quantity = @Quantity,
+                                       BaseUnit = @BaseUnit,
+                                       PurchasePrice = @PurchasePrice,
```
Comment (ngocloan202) [inline PharmacyManagement/Commodity/AllCommodities.cs:111]: format price N0 and base unit of price is VND

File: PharmacyManagement/Commodity/AllCommodities.cs [hunk 3]
```diff
@@ -1,21 +1,155 @@
...
+                                       SellingPrice = @SellingPrice,
```
Comment (ngocloan202) [inline PharmacyManagement/Commodity/AllCommodities.cs:112]: same it

**Review by ngocloan202 — CHANGES_REQUESTED** [review]

File: PharmacyManagement/Program.cs (hunk 1 above)
Comment (its-quan-ng) [reply PharmacyManagement/Program.cs]: done

File: PharmacyManagement/Commodity/AllCommodities.cs (hunk 2 above)
Comment (its-quan-ng) [reply PharmacyManagement/Commodity/AllCommodities.cs:111]: I replace it at SELECT script

File: PharmacyManagement/Commodity/AllCommodities.cs (hunk 3 above)
Comment (its-quan-ng) [reply PharmacyManagement/Commodity/AllCommodities.cs:112]: done

**Review by ngocloan202 — APPROVED** [review]

**Pull request merged**